core/ipfs/node/node.go

```diff
@@ -167,6 +159,13 @@ func (node *IpfsNode) start() error {
 }
 
 func (node *IpfsNode) stop() error {
+	node.IsRunning = false
+
+	err := node.stop()
```

Comment:
QQ: Wouldn't this cause an infinite loop? `node.stop()` calling itself with no stop condition?

Reply:
Oh thanks good catch.  Meant to call it on the inner node object. Fixed in [d4ffaef](https://github.com/FleekHQ/space-daemon/pull/219/commits/d4ffaef036f1f57511eeeaa7a53b414e7c2dea8f).